tests/host_tools/memory.py

```diff
@@ -54,7 +54,10 @@ def run(self):
         """
 
         guest_mem_bytes = self._vm.mem_size_bytes
-        ps = psutil.Process(self._vm.jailer_clone_pid)
+        try:
+            ps = psutil.Process(self._vm.jailer_clone_pid)
+        except psutil.NoSuchProcess:
```

Comment:
note for future reference: if memory monitor runs after the test was killed, the process probably ran for a very low amount of time in which case collecting the overhead doesn't really make sense.